### PR TITLE
Adapter issue 916, update installation docs

### PR DESCRIPTION
**Pull request by OsirisTerje**
See adapter issue https://github.com/nunit/nunit3-vs-adapter/issues/916

### CODE REVIEW

**Comment by SeanKilleen** [comment]
Looking at this now 👍

**Comment by SeanKilleen** [comment]
@OsirisTerje I'd love to do a pass on this whenever you think it's ready. 

It sounds like the goals are:

* Highlight the command-line based approach.
* Strongly deprecate / discourage the VSIX approach
* Ensure it's still easy to get started with the adapter

I was thinking I could throw in some screenshots etc. 

Let me know when you'd like to pass it to me and I'll work to get it over the finish line.

**Review by rprouse — APPROVED** [review]

**Comment by OsirisTerje** [comment]
@SeanKilleen  Feel free to add in those screenshots

**Comment by SeanKilleen** [comment]
Thanks! Marking this as "changes requested" so that it'll be on my radar.

**Review by SeanKilleen — CHANGES_REQUESTED** [review]

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (OsirisTerje) [reply docs/articles/vs-test-adapter/Adapter-Installation.md:7]: I don't see this text in the PR any longer?

**Comment by OsirisTerje** [comment]
@SeanKilleen Did you mean to do it as part of this PR or later ?

**Comment by SeanKilleen** [comment]
@OsirisTerje I meant to do it as part of this PR. There's some language I want to look over and make some suggestions on too. Just haven't been near a PC in a few days.

**Comment by SeanKilleen** [comment]
I apologize for the delay on this. I intend to either provide actionable feedback / changes or merge this as-is in the next day.

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:21]:
```suggestion
If you have a Visual Studio solution file in the root folder, you can go there and add the new nunit csproj:
```

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:28]: 💡 Should we add a note here about the fact that there are arguments and options to consider and running this command as-is may not produce the output folks are looking for, e.g. project name? Or should we maybe be more explicit with a comment that the project will be named after the directory?

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:35]:
```suggestion
### Manually Adding the Adapter as a Package Reference to Your Test Projects
```

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:37]:
```suggestion
When adding packages manually, we recommended installing the framework, the adapter, and the analyzer for optimal out-of-the-box functionality.  The adapter and the framework are required. The analyzer will help during development, but is not strictly required to make your project work.  `coverlet.collector` is the recommended code coverage package you should go for.
```

Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:37]: ❓ Why is the coverlet package the recommended package?

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:39]:
```suggestion
You should ensure you also reference `Microsoft.NET.Test.Sdk` as well; it is required for test discoverability.
```

I wanted to add a little more detail; check me to make sure I'm right.

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:41]:
```suggestion
1. Open your test project `csproj` file
```

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:42]:
```suggestion
2. Add the necessary package references as shown in the snippet example below (NOTE: these are the current versions at time of writing; you'll want to install the latest versions.)
```

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:54]: We should probably change this to the note format seen elsewhere in the docs. Let me know if you need a reference to where we do it elsewhere 👍

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:58]:
```suggestion
If you have a legacy project type, or prefer working outside of the command line, you can also use the Visual Studio 'Manage NuGet packages in the solution' menu option. It can also be used for the new SDK projects, but you may find it faster to open the `csproj` and copy/paste the references in.
```

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:60]:
```suggestion
With an active solution in Visual Studio, follow these steps:
```

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:62]:
```suggestion
1. From the `Tools` menu, use Library Package Manager and select `Manage NuGet packages for solution`
```

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:63]:
```suggestion
2. In the left panel, select `Online`
```

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:64]:
```suggestion
3. Locate (search for) `NUnit3Test Adapter` in the center panel and highlight it
```

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:65]:
```suggestion
4. Click `Install`
```

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:66]:
```suggestion
5. In the `Select Projects` dialog, select all test projects in your solution.
```

Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:66]: 💡 Should we add a final step of clicking OK and allowing the installation to complete? That might be helpful here.

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Installation.md:7]:
```suggestion
You'll need to install the adapter as a NuGet package for each of your test projects.
```

**Review by SeanKilleen — CHANGES_REQUESTED** [review]
@OsirisTerje apologies again for the late feedback.

I've got some requested changes around language but nothing major. I've done my best to make each item a suggestion so that you can incorporate those you agree with. 

Once the language is good to go we can merge this without the images; I can add those later and don't want to hold this up any longer than necessary.

Thanks!

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (OsirisTerje) [reply docs/articles/vs-test-adapter/Adapter-Installation.md:28]: To me it can be either way, we're not going to do a complete course on dotnet, solutions and so, - on the other hand, it doesn't cost much to add what you suggest.  So, feel free :-)

File: docs/articles/vs-test-adapter/Adapter-Installation.md
Comment (OsirisTerje) [reply docs/articles/vs-test-adapter/Adapter-Installation.md]: It is integrated into the Visual Studio Test platform as a collector, from VS version 16.5.

**Pull request merged**
